File: latihan3/php/disk.php
<?php
    class disk{
        // atribut
        private $type = "blank";
        private $capacity = 0;
        private $price = 0;
     
        // constructor
        function __construct($type="blank", $capacity=0, $price=0){
            $this->type = $type;
            $this->capacity = $capacity;
            $this->price = $price;
        }

        // Setter dan Getter

        function setType($type){
            $this->type = $type;
        }
        function getType(){
            return $this->type;
        }

        function setCapacity($capacity){
            $this->capacity = $capacity;
        }
        function getCapacity(){
            return $this->capacity;
        }

        function setPrice($price){
            $this->price = $price;
        }
        function getPrice(){
            return $this->price;
        }

        // destructor
        function __destruct(){}

    }
?>

File: latihan3/python/disk.py
class disk():
    # atribut
    __type = "blank"
    __capacity = 0
    __price = 0

    # constructor
    def __init__(self):
        self.__type = "blank"
        self.__capacity = 0
        self.__price = 0
        
        
    def __init__(self, type="blank", capacity=0, price=0):
        self.__type = type;
        self.__capacity = capacity;
        self.__price = price;

    # setter dan getter
    def setType(self, type):
        self.__type = type
        
    def getType(self):
        return self.__type 

    def setCapacity(self, capacity):
        self.__capacity = capacity
        
    def getCapacity(self):
        return self.__capacity 

    def setPrice(self, price):
        self.__price = price
        
    def getPrice(self):
        return self.__price 